### DISCUSSION TITLE
Network Drive Support for Linux

### DISCUSSION THREAD

**Discussion by zambony** [spacedriveapp/spacedrive#2702] [Feature Request]
If it isn't already planned, support for using a Windows network drive while on Linux would be helpful. On Windows, I can map a network drive easily, then add that drive as a Location, but on Linux (due to either user error or network security policies) I cannot mount some SMB shares as full folders. I can access them in Dolphin with `smb://ip/folder`, but I cannot identify a way to use that in Spacedrive.

**Comment by iLynxcat** [comment]
We should look into what could be holding us back from doing this, if it's easy we should implement @matheus-consoli 

Ideally @zambony you should be able to run Spacedrive on the server these files exist on, add the native FS volume/folder as a Location, and track that via Sync (shipping as an 0.5 feature)

**Reply by zambony** [reply]
Thanks for the reply @iLynxcat. I would run Spacedrive there but the shares are not operated by me, so I don't have control over them, only access to their contents.

**Reply by iLynxcat** [reply]
Got it. Good idea for support then. I can imagine educational users with a personal folder on a network drive in this use case. Worth exploring but I don't think this'll be an incredibly high priority item for implementation at the moment; will convert this to an issue so we can track it internally

**Reply by zambony** [reply]
That's fine, glad it got some visibility. Thank you!
